.gitignore

```diff
@@ -1,3 +1,4 @@
 .idea
 bin
 vendor
+composer.lock
```

Comment:
I'm not sure why you've changed file attributes from Unix ones to Windows ones (644 > 755).